testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -780,7 +780,7 @@ class TestPBCFlag(object):
     @dec.skipif(parser_not_found('TRZ'),
                 'TRZ parser not available. Are you using python 3?')
     def setUp(self):
-        self.prec = 3
+        self.prec = 2
```

Comment:
I find it a bit worrying that the version of numpy affects so much the simple things that are in the test class. We are talking about centre of geometry and bounding box here, not complex stuff like encore. But ok.